src/pages/Login.tsx

```diff
@@ -0,0 +1,22 @@
+import { Container, Row, Col } from "react-bootstrap"
+import LoginForm from "../components/LoginForm"
+import "./Login.css"
+
+type LoginProps = {
+  optional?: string;
+}
+
+export default (props: LoginProps) => {
+  return (
+    <Container className="d-flex align-content-between justify-content-sm-between flex-column h-100 p-5">
+      <Row className="d-flex justify-content-center mt-5 mb-5">
+        <Col md="4" className="ml-6 d-flex justify-content-sm-around" lg="4"><h1>Moneyman</h1></Col>
+      </Row>
+      <Row className="mt-5">
+        <Col className="align-self-end">
+          <LoginForm></LoginForm>
+        </Col>
+      </Row>
+    </Container>
+  )
+}
```

Comment:
Please, put a empty new line in the end of this file.
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file#:~:text=The%20empty%20line%20in%20the,can%20handle%20the%20EOF%20marker.